.github/workflows/ci.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        java: ['adopt@1.8', 'adopt@1.11', 'zulu@1.17.0-0']
+        java: [adopt@1.8, adopt@1.11, 17]
```

Comment:
I think it's better that we leave zulu here so that we know what is being used. We might want to change to adopt later along the line when it's released.

Reply:
> We might want to change to adopt later along the line when it's released

If we have it as `17`, `olafurpg/setup-scala` (transitively `jabba`) will do that for us automatically.
But if we keep it as `zulu...`, a human will have to remember to come later and make the change -- That's not worth it, IMHO. Most likely, what will happen, is that we will forget about this and will be testing against a pre-release version of Java 17 for years to come :smile_cat:

Reply:
Ach, ok cool. Didn't know that, thanks!